test/e2e/e2e-kind.sh

```diff
@@ -12,7 +12,7 @@ chmod +x kubectl && \
 sudo mv kubectl /usr/local/bin/
 
 echo ">>> Building sigs.k8s.io/kind"
-go get -u sigs.k8s.io/kind
+GO111MODULE="on" go get -u sigs.k8s.io/kind
```

Comment:
Is this related?

Reply:
Not related but without it I won't be able to merge.

Reply:
I can move it to a separate PR though.

Reply:
Is it to avoid a conflict?

Reply:
(don't worry about moving to another PR)

Reply:
> Is it to avoid a conflict?

Ah no I see, an _externality_

Reply:
Yeah, CircleCI is having a hard time anyway as it now seems to fail on the `ssh-keyscan` of bitbucket...

Reply:
@hiddeco you need to update go to 1.12.5 in e2e-golang.sh for go mod to work